### DISCUSSION TITLE
Disable Livewire 3 on specific pages.

### DISCUSSION THREAD

**Discussion by Ndinos** [livewire/livewire#8079] [1. Help] [answered]
As Livewire 3 in now embedded in Laravel and we no longer need it include `@livewireStyles` & `@livewireScripts` is there anyway of disable Livewire 3 on specific pages so that the `livewire/livewire.js` is not included on that page.

**Comment by joshhanley** [comment, marked as answer]
@Ndinos Livewire's scripts should only be injected on pages that contain a Livewire component. All other pages they shouldn't load 🙂

**Reply by Ndinos** [reply]
Thanks @joshhanley for letting me know. I thought I had removed all Livewire components, but missed one which is why I was still getting the js link.

**Reply by joshhanley** [reply]
@Ndinos no worries! 🙂 Ah yep that'll do it
